### ISSUE TITLE
CustomMode view frame issue

### ISSUE THREAD

**Issue by songjisj** [mrackwitz/MRProgress#103]
Now MRProgress forces the provided custom view to be a square if using CustomMode, related code in manualLayoutSubviews in MRProgressOverlayView:

modeViewFrame = CGRectMake(modePadding, y, innerViewWidth, innerViewWidth);

Could you use the height which the custom view provides? Suggested change like:

if (self.mode == MRProgressOverlayViewModeCustom) {
                modeViewFrame = CGRectMake(modePadding, y, innerViewWidth, self.modeView.frame.size.height);
            } 

Thank you.

**Comment by mrackwitz** [comment]
You're right. Using the provided height would make much more sense here. Are you eager to put together a PR for that? That would be welcome.
